pytorch_lightning/accelerators/dp_backend.py

```diff
@@ -65,17 +67,17 @@ def __init_torch_data_parallel(self, model):
         model = LightningDataParallel(model, device_ids=device_ids)
         return model
 
-    def __init_half_precision(self, model):
+    def _setup_half_precision(self, model):
```

Comment:
i'm curious, why the naming change? the pattern here seems to be `__init` everywhere?

Reply:
I changed it before rebasing the refactors to match with other backend APIs, but while rebasing it never showed up in conflict. Reverted it. Thanks again.